# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1319487254 14400
#      Mon Oct 24 16:14:14 2011 -0400
# Branch stable
# Node ID eac43686d7916be7704e40112cab59115529fa65
# Parent  50ef82665546938bd5c8d80e02d392bc76d44a3d
maint: missing semicolon.

* __axes_limits__.m: Missing semicolon.

diff --git a/scripts/plot/private/__axes_limits__.m b/scripts/plot/private/__axes_limits__.m
--- a/scripts/plot/private/__axes_limits__.m
+++ b/scripts/plot/private/__axes_limits__.m
@@ -40,17 +40,17 @@ function retval = __axes_limits__ (fcn, 
       elseif (strcmpi (arg, "auto") ||  strcmpi (arg, "manual"))
         set (h, fcnmode, arg);
       endif
     else
       if (!isnumeric (arg) && any (size(arg(:)) != [2, 1]))
         error ("%s: argument must be a 2 element vector", fcn);
       else
         if (arg(1) >= arg(2))
-          error ("%s: axis limits must be increasing", fcn)
+          error ("%s: axis limits must be increasing", fcn);
         else
           set (h, fcn, arg(:));
         endif
       endif
     endif
   endif
 
 endfunction
